accounting/acct.proto

```diff
@@ -41,6 +41,35 @@ import "github.com/openconfig/gnoi/types/types.proto";
 option go_package = "github.com/openconfig/gnsi/accounting";
 option (gnoi.types.gnoi_version) = "0.2.0";
 
+//
+// The Accounting service describes the interface between a system
+// and downstream management systems that may want/need to collect
+// data about what changes were attempted or completed on that system.
+// 
+// Accounting messages may be collected by connecting to the Accounting
+// service on a system from a remote endpoint, or by configuring a
+// system to connect to the remote endpoint directly.
+//
+// Records created on a system can be delivered to remote collectors,
+// stored locally, or discarded locally. Configuration of this behavior
+// is done with standard gNMI system configuration methods.
+
+service Accounting {
```

Comment:
ok, I guess I can see why 2 services makes sense, part of that is in a comment I made about: "this is expected to be implemented on the remote collector...." (and it'd be confusing to have a dangling service or unimplemented stub)

The point about flushing.. that's just TCP doing it's thing here, right? Messages queue up on the sender if the tcp window isn't large enough or there's application congestion on the far end... if you let tthat queue get too large the configured limits at the source are just going to trim the queue for you. That's sad, because you'll lose records, but also cpu is cheap so stop doing this on your 8086 collector?

I'm certain I mis-understood your second point though :)

Reply:
how's that look to you? (I didn't see a reason to change the rpc definitions so much, but happy to adjust if necessary.

Reply:
From chat I think we need to be clear about:
  1) refer to size of cache constraints
  2) send sequence numbers and expect ack's every X of those - stream is 1 record per send, 1 ACK per N records received
  3) on connect offer 'last seen'  sequence number

Reply:
i think these three items are done.